docs/rest_api_guide/03_RESTstatistics.rst

```diff
@@ -0,0 +1,177 @@
+    * - PURGED
+      - Readings removed from the buffer by the purge process
+    * - READINGS
+      - Readings received by Fledge
+    * - UNSENT
+      - Readings filtered out in the send process
+    * - UNSNPURGED
+      - Readings that were purged from the buffer before being sent
+
+In addition to these fixed names there will be;
+
+  - One statistic per north service of task that is named the same as the service or task name. This will count the number of readings sent out on that service.
+
+  - One statistic per asset that is named the same as the asset. This will be the number of readings that have been ingested for that asset.
+
+  - One statistics per south service, that is named with the service name and *-Ingest* appended. This is the count of readings read in for that service.
+
+GET statistics
+~~~~~~~~~~~~~~
+
+``GET /fledge/statistics`` - return a general set of statistics
+
+
+**Response Payload**
+
+The response payload is a JSON document with statistical information (all numerical), these statistics are absolute counts since Fledge started.
+
+
+**Example**
+
+.. code-block:: console
+
+  $ curl -s http://localhost:8081/fledge/statistics
+  [ {
+      "key": "BUFFERED",
+      "description": "Readings currently in the Fledge buffer",
+      "value": 0
+    },
+  ...
+    {
+      "key": "UNSNPURGED",
+      "description": "Readings that were purged from the buffer before being sent",
+      "value": 0
+    },
+  ... ]
+  $
+
+
+GET statistics/history
+~~~~~~~~~~~~~~~~~~~~~~
+
+``GET /fledge/statistics/history`` - return a historical set of statistics. This interface is normally used to check if a set of sensors or devices are sending data to Fledge, by comparing the recent statistics and the number of readings received for an asset.
+
+
+**Request Parameters**
+
+- **limit** - limit the result set to the *N* most recent entries.
+
+
+**Response Payload**
+
+A JSON document containing an array of statistical information, these statistics are delta counts since the previous entry in the array. The time interval between values is a constant defined that runs the gathering process which populates the history statistics in the storage layer.
+
+.. list-table::
+    :widths: 20 50
+    :header-rows: 1
+
+    * - Key
+      - Description
+    * - interval
+      - The interval in seconds between successive statistics values
+    * - statistics[].BUFFERED
+      - Readings currently in the Fledge buffer
+    * - statistics[].DISCARDED
+      - Readings discarded by the South Service before being  placed in the buffer. This may be due to an error in the readings themselves.
+    * - statistics[].PURGED
+      - Readings removed from the buffer by the purge process
+    * - statistics[].READINGS
+      - Readings received by Fledge
+    * - statistics[].*NORTH_TASK_NAME*
+      - The number of readings sent to the PI system via the OMF plugin with north instance name
```

Comment:
`PI system via the OMF plugin` is one of an example.
As this can be true for any north configured